### PR TITLE
NETOBSERV-1832 add critical netobserv tests in all CI runs

### PR DESCRIPTION
**Pull request by memodi**
we would like to include Critical netobserv tests to run in all CI profiles, `Network_Observability.*Critical` pattern will only capture below tests:

```
$ bin/extended-platform-tests run --dry-run all | grep "Network_Observability.*Critical"
"[sig-netobserv] Network_Observability FLP and Console metrics: when processor.metrics.TLS == Auto Author:aramesha-LEVEL0-Critical-54043-Critical-66031-Verify flowlogs-pipeline and Console metrics [Serial]"
"[sig-netobserv] Network_Observability FLP and Console metrics: when processor.metrics.TLS == Disabled Author:aramesha-LEVEL0-Critical-50504-Verify flowlogs-pipeline metrics and health [Serial]"
"[sig-netobserv] Network_Observability eBPF metrics when agent.ebpf.metrics.TLS == Disabled Author:aramesha-LEVEL0-Critical-72959-Verify eBPF agent metrics and health [Serial]"
```

### CODE REVIEW

**Comment by memodi** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-gcp-ipi-marketplace-f28-destructive

**Comment by memodi** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-gcp-ipi-marketplace-f28

**Comment by memodi** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-gcp-ipi-marketplace-f28-destructive

**Comment by memodi** [comment]
@kuiwang02 @jianzhangbjz this PR is ready to be merged could you PTAL?

**Comment by kuiwang02** [comment]
I am OK for step change.

please request qe-producitivity team to approve it for regression job change.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml [hunk 1]
```diff
@@ -1995,11 +1992,8 @@ tests:
   steps:
     cluster_profile: equinix-ocp-metal-qe
     env:
-      DEVSCRIPTS_CONFIG: |
-        FIPS_MODE=true
-        IP_STACK=v4
-        NETWORK_TYPE=OVNKubernetes
-      TEST_ADDITIONAL: Network_Observability
+      DEVSCRIPTS_CONFIG: "FIPS_MODE=true\nIP_STACK=v4\nNETWORK_TYPE=OVNKubernetes
+        \     \n"
```
[the remark this answers is not in the document]
Comment (memodi) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml]: I think make targets just auto formatted, reverted it now.

**Pull request merged**